numba/core/typed_passes.py

```diff
@@ -352,10 +353,10 @@ def run_pass(self, state):
         return True
 
 
-@register_pass(mutates_CFG=True, analysis_only=False)
-class NativeLowering(LoweringPass):
+class BaseNativeLowering(LoweringPass):
 
-    _name = "native_lowering"
+    _name = None
+    _lowering_class = None
```

Comment:
Maybe making this abstractclassmethod so we lock down how it can be modified a little bit.